<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditarQuestoes</class>
 <widget class="QDialog" name="EditarQuestoes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Query questoes:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="etSelectTable"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="questoesTable">
     <property name="minimumSize">
      <size>
       <width>311</width>
       <height>0</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NomeTopico</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nivel</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LimiteTempo (s)</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>547</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Limite de Tempo: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nível:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="etNome"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="etSaida"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="etTempo"/>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="etUpdate"/>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="cbTopico"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="etId"/>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Query Remove Resolve: </string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Query Remove Propoe: </string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Adicionar Questão</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="etEnunciado"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="etNivel"/>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Atualizar Questão</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="etBusca"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Query Remove Possui: </string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Saida:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Entrada:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nome:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Enunciado:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Nome_Topico:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="etEntrada"/>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="etAdiciona"/>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Buscar Questão</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Query Remove Questao: </string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="etRemovePossui"/>
       </item>
       <item row="13" column="1">
        <widget class="QLineEdit" name="etRemovePropoe"/>
       </item>
       <item row="14" column="1">
        <widget class="QLineEdit" name="etRemoveResolve"/>
       </item>
       <item row="15" column="1">
        <widget class="QLineEdit" name="etRemoveQuestao"/>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Remover Questão</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Questões ativas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>EditarQuestoes</receiver>
   <slot>buscar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>EditarQuestoes</receiver>
   <slot>atualizar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditarQuestoes</receiver>
   <slot>adicionar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>EditarQuestoes</receiver>
   <slot>remover()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buscar()</slot>
  <slot>adicionar()</slot>
  <slot>atualizar()</slot>
  <slot>remover()</slot>
 </slots>
</ui>
